test/irmin-pack/test_existing_stores.ml

```diff
@@ -416,6 +416,65 @@ module Test_corrupted_stores = struct
     S.Repo.close rw >>= fun () -> S.Repo.close ro
 end
 
+module Test_corrupted_inode = struct
+  let root_archive, root =
+    let open Fpath in
+    ( v "test" / "irmin-pack" / "data" / "corrupted_inode" |> to_string,
```

Comment:
I'm curious. How did you generate this file? :D

Reply:
I generated it on branch not containing the fix for remove (https://github.com/icristescu/irmin/commit/384aea20c5540413aa00403d4ff73e0aab20de18)

Reply:
👍